src/tribol/integ/Integration.cpp

```diff
@@ -37,65 +38,98 @@ TRIBOL_HOST_DEVICE void EvalWeakFormIntegral< COMMON_PLANE, SINGLE_POINT >
                         cx[0], cx[1], cx[2] );
    }
 
-   ///////////////////////////////////////////////
-   //
-   //project overlap polygon centroid to each face
-   //
-   ///////////////////////////////////////////////
-   RealT cxProj1[3]  = { 0., 0., 0. }; // overlap centroid projected to face 1
-   RealT cxProj2[3]  = { 0., 0., 0. }; // overlap centroid projected to face 2
-   RealT cxf1[3] = { 0., 0., 0. }; // vertex avg. centroid of face 1
-   RealT cxf2[3] = { 0., 0., 0. }; // vertex avg. centroid of face 2
-
-   // compute vertex averaged centroid of each face for the point-normal data
-   VertexAvgCentroid( elem.faceCoords1, elem.dim, elem.numFaceVert,
-                      cxf1[0], cxf1[1], cxf1[2] );
-   VertexAvgCentroid( elem.faceCoords2, elem.dim, elem.numFaceVert,
-                      cxf2[0], cxf2[1], cxf2[2] );
-
-   // project the overlap centroid to each face
+   // debug: leave commented out so we don't enter loop
```

Comment:
If we had an `#ifdef TRIBOL_DEBUG`, this would be a good place to use it.

Reply:
Definitely. I think we can address this when we do our next overhaul of logging such that debug loops are properly guarded instead of commented out.